web_timeline/static/lib/vis/vis.css

```diff
@@ -0,0 +1,810 @@
+.vis .overlay {
```

Comment:
I know it's common to include the libraries in the source code, but [given this one is npm-installable](http://visjs.org/index.html#download_install), it would be much nicer IMHO [if you add a `package.json` file that declares the dependencies](https://docs.npmjs.com/files/package.json#dependencies) and add to installation instructions that you should execute `npm install` after downloading the module to make it work. OTOH, you could even execute `cd static/src/js && npm list vis || npm install vis` manually in the `pre_init` script, and provide a useful exception if you have no permissions to write it and it was not found.

Of course, package would be now under `static/src/js/node_modules`.